docs/developer-guide/integrations/users/geoserver.md

```diff
@@ -142,7 +146,56 @@ The last step is to configure MapStore to use the authkey with the configured in
  - Verify that "useAuthenticationRules" is set to `true`
  - `authenticationRules` array should contain 2 rules:
      - The first rule should already be present, and defines the authentication method used internally in mapstore
-     - The second rule (the one you need to add) should be added and defines how to autenticate to GeoServer:
+     - The second rule (the one you need to add) should be added and defines how to authenticate to GeoServer:
          - `urlPattern`: is a regular expression that identifies the request url where to apply the rule
          - `method`: set it to `authkey` to use the authentication filter you just created in Geoserver.
          - `authkeyParamName`: is the name of the authkey parameter defined in GeoServer (set to `authkey` by default)
+
+## Other geoserver integration
+
+You can allow some users to:
+
+- Execute some processes (via [WPS security](http://gs-stable.geo-solutions.it/geoserver/web/wicket/bookmarkable/org.geoserver.wps.web.WPSAccessRulePage?25&filter=false)) 
+
+  For example download feature can leverage gs:download process to allow user to download data
+
+- Edit Styles (by default allowed only to administrators, but you can change it acting on /rest/ Filter Chains). 
+
+- Access to layers based on users (standard geoserver security)
+
+- Filter layers data based on users (GeoFence), see [here](https://docs.geoserver.org/latest/en/user/extensions/geofence-server/index.html)
+
+- Allow editing of layers to certain MapStore users ([GeoServer Security](https://docs.geoserver.org/stable/en/user/security/webadmin/data.html)). The editing can be enabled in the [plugin settings of MapStore](https://dev-mapstore.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins.FeatureEditor)
+
+## GeoServer Plugins and Extensions
+
+MapStore supports several plugins for GeoServer
+
+- [WMTS Multidimensional](https://docs.geoserver.org/stable/en/user/community/wmts-multidimensional/index.html) despite the name, this service provides multidimensional discovery services for GeoServer in general, and is **required** to use the timeline plugin of MapStore.
+
+- [SLD Rest Service](https://docs.geoserver.org/latest/en/user/extensions/sldservice/index.html) Is an extension that can be used by the MapStore styler to classify Vector and Raster data. It can inspect the real layer data to apply classification based on values contained in it. It allows to select various classification types (quantile, equalInterval, standardDeviation…) and to customize the color scales based on parameters
```

Comment:
```suggestion
- [SLD Rest Service](https://docs.geoserver.org/latest/en/user/extensions/sldservice/index.html) is an extension that can be used by the MapStore styler to classify Vector and Raster data. It can inspect the real layer data to apply classification based on values contained in it. It allows to select various classification types (quantile, equalInterval, standardDeviation…) and to customize the color scales based on parameters
```